test/jdk/javax/net/ssl/templates/SSLExampleCert.java

```diff
@@ -359,18 +368,41 @@ private static SSLContext createSSLContext(
             }
         }
 
+        // Set the date for the verifying of certificates.
+        @SuppressWarnings("deprecation")
+        Date verifyingDate = new Date(123, 3, 3);
```

Comment:
can DateFormat be used instead to parse Date from a readable string representation?

Reply:
Good catch!  Thank you!